### PR TITLE
[runtime-security] Package default policy from DataDog/security-agent-policies

### PR DESCRIPTION
**Pull request by lebauce**
### What does this PR do?

Package the default policy fetch from the DataDog/security-agent-policies repository

### CODE REVIEW

**Review by safchain — APPROVED** [review]

File: pkg/security/config/default.go [hunk 1]
```diff
@@ -7,56 +7,109 @@ package config
 
 // DefaultPolicy holds the default runtime security agent rules
 var DefaultPolicy = `---
-version: 1.0.0
+version: 1.0.1
```
Comment (clamoriniere) [inline pkg/security/config/default.go:10]: In the Pull request description you said

> Package the default policy fetch from the DataDog/security-agent-policies repository

But here it looks like also add a default policy directly integrated to the security-agent binary.

Comment (lebauce) [reply pkg/security/config/default.go:10]: Yes, this is temporary. It was decided to use the policies from the `DataDog/security-agent-policies` but we still want to test the default policy loads fine. We'll add ci-jobs on `security-agent-policies` that will use the latest build of the agent and test. Does it make sense ?

Comment (safchain) [reply pkg/security/config/default.go:10]: We use this for testing purpose. We will remove this once we will have test/CI job on the policies repo side

File: omnibus/config/software/datadog-security-agent-policies.rb [hunk 2]
```diff
@@ -38,4 +38,22 @@
 
   end
 
+  if linux?
```
Comment (KSerrania) [inline omnibus/config/software/datadog-security-agent-policies.rb:41]: Nit: this `if` is not needed (at least for now), as this software definition is only added on linux: https://github.com/DataDog/datadog-agent/blob/b05aa49c616c8872766154e2de71b26608885d58/omnibus/config/projects/agent.rb#L205-L207

**Review by KSerrania — APPROVED** [review]
The omnibus change looks good to me.

File: omnibus/config/software/datadog-security-agent-policies.rb (hunk 2 above)
Comment (lebauce) [reply omnibus/config/software/datadog-security-agent-policies.rb]: Indeed. Fixed

**Pull request merged**